src/js/bootstrap-slider.js

```diff
@@ -1453,7 +1453,7 @@ const windowIsDefined = (typeof window === "object");
 				this._setDataVal(newValue);
 				this.setValue(newValue, false, true);
 
-				this._pauseEvent(ev);
+				ev.returnValue = false;
```

Comment:
No, you can't do like that. Look in function _pauseEvent =>
```
_pauseEvent: function(ev) {
				if(ev.stopPropagation) {
					ev.stopPropagation();
				}
				if(ev.preventDefault) {
					ev.preventDefault();
				}
				ev.cancelBubble=true;
				ev.returnValue=false;
			}
```
the three commands stop bubbling and the last one not. The last command (`ev.returnValue=false;`) helps to stop slider-handle bee active, so there will be no outline on handle after we grab it. In fact, we need only the last one command in the `_mousedown` event handler. I didn't find any issues in different browsers (except safari). 
You can experiment with it in jsfiddle: https://jsfiddle.net/zevs7772006/zzcb14m9/1/ 
You need the 1456 - 1457 lines
```
//ev.returnValue=false;
this._pauseEvent(ev);
```
try to lunch modal and drag the slider  handler off the modal.
then uncomment 1456 line and comment 1457 line and see the difference.